compass/ocean/tests/baroclinic_channel/default/__init__.py

```diff
@@ -47,3 +48,12 @@ def configure(self):
         baroclinic_channel.configure(self.resolution, self.config)
 
     # no run() is needed because we're doing the default: running all steps
+
+    def validate(self):
+        """
+        Test cases can override this method to perform validation of variables
```

Comment:
```suggestion
        Compare ``temperature``, ``salinity``, ``layerThickness`` and ``normalVelocity`` with a baseline if one was provided.
```
I realize the test case you copied from probably don't have a useful docstring here but we probably should fix that here.

Reply:
Make sure the line if under 79 characters.  That's hard for me to tell in GitHub.